cli/src/commands/pull/pull.test.ts

```diff
@@ -217,6 +217,15 @@ const pgrollFetchSingle = (url: string, request: any) => {
     return baseFetch(url, request);
   }
 };
+const pgrollInferredMigration = {
+  done: true,
+  migration:
+    '{"name": "mig_abcdcdrj7c92neg7lefg", "operations": [{"sql": { "up": "ALTER ALTER TABLE internal ADD test varchar(255)" }}]}',
```

Comment:
This reminds me that we discussed to JSON.parse the migration and expand the object here for readability 😄

Reply:
Ah good point. Doing it now so I don't forget